controllers/provisioning_controller.go

```diff
@@ -497,9 +493,9 @@ func (r *ProvisioningReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	if enabled {
 		baremetalConfig, err := r.readProvisioningCR(context.Background())
 		if err != nil || baremetalConfig == nil {
-			err = r.updateCOStatus(ReasonComplete, "Provisioning CR not found on BareMetal Platform; marking operator as available", "")
+			err = r.updateCOStatus(ReasonProvisioningCRNotFound, "Provisioning CR not found on BareMetal Platform", "")
 			if err != nil {
-				return fmt.Errorf("unable to put %q ClusterOperator in Available state: %w", clusterOperatorName, err)
+				return fmt.Errorf("unable to put %q ClusterOperator in Default state: %w", clusterOperatorName, err)
```

Comment:
Can we get this error msg to match what is in https://github.com/openshift/cluster-baremetal-operator/pull/191/files#diff-7ad72d08a06fe28b03a15c84289b815f693443562d0c67e6f461c7e3f5ee0707R195?